packages/shared-ux/chrome/navigation/tsconfig.json

```
@@ -0,0 +1,22 @@
+{
+  "extends": "../../../../tsconfig.base.json",
+  "compilerOptions": {
+    "outDir": "target/types",
+    "types": [
+      "jest",
+      "node",
+      "react"
```

Add the first for `emotion`, and the second to include ambient types, (like support for importing `.mdx` files)

```suggestion
      "react",
      "@emotion/react/types/css-prop",
      "@kbn/ambient-ui-types"
```